contracts/factories/RewardFactory.sol

```diff
@@ -10,8 +10,6 @@ import "../utils/MathUtil.sol";
 contract RewardFactory is IRewardFactory {
     using MathUtil for uint256;
 
-    event ExtraRewardAdded(address reward, uint256 pid);
```

Comment:
Hmm, why do you think this code is not going to be used?

Reply:
It is related to ExtraRewardStashV2, and we are using v3, which doesn't need this anymore. And it is not used anywhere